idtoken/idtoken_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 Google LLC.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package idtoken
```

Comment:
I think this test should either be removed for now or else we need to turn it into a real integration test and setup creds. As is this is causing CI to fail.